traffic_ops/app/db/create_tables.sql

```diff
@@ -462,6 +534,54 @@ ALTER TABLE cdn_id_seq OWNER TO traffic_ops;
 
 ALTER SEQUENCE cdn_id_seq OWNED BY cdn.id;
 
+--
+-- Name: cdn_lock; Type: TABLE; Schema: public; Owner: traffic_ops
+--
+
+CREATE TABLE IF NOT EXISTS public.cdn_lock (
+    username text NOT NULL,
+    cdn text NOT NULL,
+    message text,
+    soft boolean NOT NULL DEFAULT TRUE,
+    last_updated timestamp with time zone DEFAULT now() NOT NULL,
+    CONSTRAINT pk_cdn_lock PRIMARY KEY ("cdn")
+);
+
+
+ALTER TABLE cdn_lock OWNER TO traffic_ops;
+
+--
+-- Name: cdn_notification; Type: TABLE; Schema: public; Owner: traffic_ops
+
+CREATE TABLE cdn_notification (
+    id bigint NOT NULL,
+    cdn text NOT NULL,
+    "user" text NOT NULL,
+    notification text,
+    last_updated timestamp with time zone DEFAULT now() NOT NULL,
+    CONSTRAINT pk_cdn_notification PRIMARY KEY (id)
```

Comment:
Ah, I see what happened. The constraint is explicitly named as `pk_cdn_notification` in the original migration (`2021021900000000_cdn_notification.up.sql`) that created the table, and the subsequent migration that re-created it didn't specify a name, so I didn't notice it may have changed.